client/app/scripts/superdesk-publish/publish.js

```diff
@@ -283,7 +283,13 @@
             );
         };
 
-        $scope.filterSchedule = function() {
+        $scope.filterSchedule = function(item, type) {
+            if (type === 'channel') {
+                $scope.selectedFilterChannel = item;
+            } else if (type === 'subscriber') {
+                $scope.selectedFilterSubscriber = item;
+            }
```

Comment:
Well, you still need to test that this little change indeed assigns values to predefined variables. :wink: